test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js

```diff
@@ -1421,23 +1141,28 @@ describe('test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js', () => {
             headObject: async (params) => generateMatchingHeadObjectResponse(serverless, params),
```

Comment:
`generateMatchingListObjectsResponse` and `generateMatchingHeadObjectResponse` functions has some TODO parts not addressed.

To have this test fully complete either we have to update those functions, or if there's no need to rely on them in this case, we can simply put payload inline.

Reply:
Function are now completed.